packages/ndk/pubspec.yaml

```diff
@@ -0,0 +1,35 @@
+name: ndk
+description: Nostr Development Kit - the most performant lib for all your nostr usecases
+version: 0.1.3
+homepage: https://github.com/relaystr/ndk
+
+environment:
+  sdk: ">=3.4.0 <4.0.0"
+
+platforms:
+  android:
+  ios:
+  linux:
+  macos:
+  web:
+  windows:
+
+dependencies:
+  http: ^1.2.2
+  bip340: ^0.3.0
+  bech32: ^0.2.2
+  hex: ^0.2.0
+  web_socket_channel: ^3.0.1
+  crypto: ^3.0.5
+  collection: ^1.18.0
+  convert: ^3.1.1
+  pointycastle: ^3.9.1
+  logger: ^2.4.0
+  rxdart: ^0.28.0
+  isar: ^4.0.0-dev.14
+
+dev_dependencies:
+  build_runner: ^2.4.11
+  mockito: ^5.0.17
+  objectbox_generator: any
```

Comment:
possible to remove?

Reply:
objectbox_generator yes,
mockito is needed for sure,
build_runner I'm not sure, it is used for generating isar when changing something in the db models